README.md

```diff
@@ -6,6 +6,7 @@ products.
 ## Available charts
 
 * Java [Advanced Management Console](./java-amc) (AMC)
+* Oracle SOA Suite [chart](./soa-suite)
```

Comment:
Could you please link all of `Oracle SOA Suite` and remove `chart`? It's implied. :)